knotx-standalone/config/application.conf

```diff
@@ -1,12 +1,103 @@
+########### Modules to start ###########
 modules = [
-    "server=io.knotx.server.KnotxServerVerticle"
-    "httpRepo=io.knotx.repository.http.HttpRepositoryConnectorVerticle"
-    "fsRepo=io.knotx.repository.fs.FilesystemRepositoryConnectorVerticle"
-    "splitter=io.knotx.splitter.FragmentSplitterVerticle"
-    "assembler=io.knotx.assembler.FragmentAssemblerVerticle"
-    "hbsKnot=io.knotx.knot.templating.HandlebarsKnotVerticle"
-    "serviceKnot=io.knotx.knot.service.ServiceKnotVerticle"
-    "actionKnot=io.knotx.knot.action.ActionKnotVerticle"
-    "serviceAdapter=io.knotx.adapter.service.http.HttpServiceAdapterVerticle"
-  ]
+  "server=io.knotx.server.KnotxServerVerticle"
+  "httpRepo=io.knotx.repository.http.HttpRepositoryConnectorVerticle"
+  "splitter=io.knotx.splitter.FragmentSplitterVerticle"
+  "assembler=io.knotx.assembler.FragmentAssemblerVerticle"
+  "hbsKnot=io.knotx.knot.templating.HandlebarsKnotVerticle"
+  "serviceKnot=io.knotx.knot.service.ServiceKnotVerticle"
+  # Enable action knot if you want to have support for form submittions. Follow the actionKnot configuration to set it up
+  # "actionKnot=io.knotx.knot.action.ActionKnotVerticle"
```

Comment:
Action Knot is disabled but configured in the routing.

Reply:
Yes, I'm thinking actually about enabling actionKnot and leaving the config as it's now - and clarify in the config that if you're not using any module, just comment it out or remove.